_dev/js/back/src/App.vue

```diff
@@ -0,0 +1,257 @@
+      </template>
+      <MenuItem route="/help">
+        {{ $t("menu.help") }}
+      </MenuItem>
+    </Menu>
+
+    <div class="pt-5" />
+    <div class="pt-3" />
+
+    <div class="container" v-if="isShopContext">
+      <RoundingBanner />
+    </div>
+
+    <div class="container" v-if="isTestMode">
+      <b-alert variant="warning" show>
+        <p>{{ $t("general.testModeOn") }}</p>
+      </b-alert>
+    </div>
+
+    <div class="container" v-if="!isShopContext">
+      <b-alert variant="warning" show>
+        <h2>{{ $t("general.multiShop.title") }}</h2>
+        <p>{{ $t("general.multiShop.subtitle") }}</p>
+        <p>{{ $t("general.multiShop.chooseOne") }}</p>
+        <b-list-group
+          v-for="group in shopsTree"
+          :key="group.id"
+          class="mt-3 mb-3 col-4"
+        >
+          <p class="text-muted">
+            {{ $t("general.multiShop.group") }} {{ group.name }}
+          </p>
+          <b-list-group-item
+            v-for="shop in group.shops"
+            :key="shop.id"
+            :href="shop.url"
+          >
+            {{ $t("general.multiShop.configure") }} <b>{{ shop.name }}</b>
+          </b-list-group-item>
+        </b-list-group>
+        <p>{{ $t("general.multiShop.tips") }}</p>
+      </b-alert>
+    </div>
+
+    <router-view v-if="isShopContext" />
+  </div>
+</template>
+
+<script>
+import Menu from "@/components/menu/menu";
+import MenuItem from "@/components/menu/menu-item";
+import RoundingBanner from "@/components/block/rounding-banner";
+
+export default {
+  name: "Home",
+  components: {
+    Menu,
+    MenuItem,
+    RoundingBanner
+  },
+  data() {
+    return {
+      paypalStatusUpdater: null
+    };
+  },
+  methods: {
+    updater() {
+      this.paypalStatusUpdater = setInterval(() => {
+        this.$store.dispatch("refreshPaypalStatus");
+      }, 10000);
+    }
+  },
+  computed: {
+    onboardingPaypalIsCompleted() {
+      return this.$store.state.paypal.onboardingCompleted;
+    },
+    onboardingFirebaseIsCompleted() {
+      return this.$store.state.firebase.onboardingCompleted;
+    },
+    accountIslinked() {
+      return this.$store.state.paypal.accountIslinked;
+    },
+    isShopContext() {
+      return this.$store.state.context.isShopContext;
+    },
+    isTestMode() {
+      return (
+        this.$store.state.configuration.paymentMode === "SANDBOX" &&
+        this.$store.state.context.isShopContext
+      );
+    },
+    shopsTree() {
+      return this.$store.state.context.shopsTree;
+    }
+  },
+  watch: {
+    accountIslinked(val) {
+      if (!val && this.onboardingPaypalIsCompleted) {
+        this.updater();
+      } else {
+        clearInterval(this.paypalStatusUpdater);
+      }
+    }
+  },
+  created() {
+    if (!this.onboardingPaypalIsCompleted || this.accountIslinked) {
+      return;
+    }
+
+    this.updater();
+  }
+};
+</script>
+
+<style lang="scss">
+#app {
+  @import "../../../../node_modules/bootstrap-vue/dist/bootstrap-vue";
```

Comment:
Isnt it possible to do something like this ? 

```suggestion
  @import "~bootstrap-vue/dist/bootstrap-vue";
```

Reply:
I don't know, I try but I have an error : `Cannot resolve directory '~'` 🙃

Reply:
Ah, looks like Webpack doesn't recognize npm dependencies, I'm gonna take a look